app/services/fandom-com-migration.js

```diff
@@ -0,0 +1,68 @@
+import Service, { inject as service } from '@ember/service';
```

Comment:
how about putting it in ember-fandom? looks like something common for mobile-wiki, discussions-front-end and announcements-front-end

Reply:
We'll show it only in mobile-wiki and oasis.